packages/sdk/README.md

```
@@ -85,149 +84,72 @@ Get your `projectId` and `apiKey` from your [Bklit Dashboard](https://app.bklit.
+- `🚀 Bklit SDK: Message sent via WebSocket`
+- `📦 Bklit SDK: Message queued (WebSocket not ready)`
+- `🔄 Bklit SDK: Reconnecting...`
 
-Clear the current session (useful for testing).
+Error messages (using `console.error` and `console.warn`) always appear regardless of the debug setting.
 
-**Returns:** void
+## Architecture
 
-## Support
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)